src/device/wg_controller.rs

```diff
@@ -472,9 +473,15 @@ async fn build_requested_peers_list<
             // Exit node is a fresh node, therefore - insert create new peer
             let public_key = exit_node.public_key;
             let endpoint = exit_node.endpoint;
+
+            let ip_addresses = vec![
+                IpAddr::V4(Ipv4Addr::from(VPN_INTERNAL_IPV4)),
```

Comment:
So if we are moving with internal check I think we can just lveve the internal ips.

Also  ipv6 address should only be added if ipv6 feature is enabled.

Reply:
We either try to ping all of the items in the ip_address range, or just leave the ones the dual target will take and additionally ping external ip in qos on failure.